docs/relational-databases/data-compression/row-compression-implementation.md

```diff
@@ -47,7 +47,7 @@ monikerRange: "=azuresqldb-current||>=sql-server-2016||>=sql-server-linux-2017||
 |**money**|Yes|Uses the integer data representation by using an 8-byte integer. Currency value is multiplied by 10000 and the resulting integer value is stored by removing any digits after the decimal point. This type has a larger range than **smallmoney**. This type has a storage optimization similar to that for integer types.|  
 |**float**|Yes|Least significant bytes with zeros are not stored. **float** compression is applicable mostly for nonfractional values in mantissa.|  
 |**real**|Yes|Least significant bytes with zeros are not stored. **real** compression is applicable mostly for nonfractional values in mantissa.|  
-|**smalldatetime**|No|Uses the integer data representation by using two 2-byte integers. The date takes 2 bytes. It is the number of days since 1/1/1901. This needs 2 bytes starting from 1902. Therefore, there is no savings after that point.<br /><br /> The time is the number of minutes since midnight. Time values that are slightly past 4AM start to use the second byte.<br /><br /> If a **smalldatetime** is only used to represent a date (a common case), the time is 0.0. Compression saves 2 bytes by storing the time in most significant byte format for row compression.|  
+|**smalldatetime**|No|Uses the integer data representation by using two 2-byte integers. The date takes 2 bytes. It is the number of days since 1900-01-01. This needs 2 bytes starting from 1900-09-14. Therefore, there is no savings after that point.<br /><br /> The time is the number of minutes since midnight. Time values that are slightly past 4AM start to use the second byte.<br /><br /> If a **smalldatetime** is only used to represent a date (a common case), the time is 0.0. Compression saves 2 bytes by storing the time in most significant byte format for row compression.|  

```

Comment:
@WilliamDAssafMSFT ,

I can't do it in the next several hours because of another commitment but would you mind if I did a test to make sure that's true?

Reply:
@JeffModen Of course! No hurry.